packages/docs/PropTables/StatefulTablePropTable.tsx

```diff
@@ -170,11 +176,6 @@ const filterProps: Prop[] = [
     name: 'pillTabs',
     types: 'PillTabItem[]',
   },
-  {
```

Comment:
Is this an intended delete?

Reply:
Oh, no 🙈
Thanks for your attentiveness!